src/GeneralStateTestsFiller/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalidFiller.json

```diff
@@ -0,0 +1,275 @@
+                        "code": "0x"
+                    },
+                    "82a978b3f5962a5b0957d9ee9eef472ee55b42f1": {
+                        "storage": {},
+                        "nonce": "7",
+                        "code": "0x"
+                    },
+                    "0000000000000000000000000000000000000006": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1",
+                        "code": "0x"
+                    },
+                    "0000000000000000000000000000000000000007": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1",
+                        "code": "0x"
+                    },
+                    "598443f1880ef585b21f1d7585bd0577402861e5": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "7d577a597b2742b498cb5cf0c26cdcd726d39e6e": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "0000000000000000000000000000000000000004": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1",
+                        "code": "0x"
+                    },
+                    "e0fc04fa2d34a66b779fd5cee748268032a146c0": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "0000000000000000000000000000000000000002": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1",
+                        "code": "0x"
+                    },
+                    "77db2bebba79db42a978f896968f4afce746ea1f": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "10a1c1cb95c92ec31d3f22c66eef1d9f3f258c6b": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "90f0b1ebbba1c1936aff7aaf20a7878ff9e04b6c": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    }
+                }
+            }
+        ],
+        "explanation": "Puts the given data into the ECPAIRING precompile"
```

Comment:
we agreed to use //for comments in the fillers
and if you want the comment to end up in generated test fill the description into
```
"_info" : {
            "comment" : ""
}
```

_info.comment should be copied to the final test when filling

Reply:
OK.